package.json

```diff
@@ -70,6 +74,6 @@
     "@types/node": "^22.13.4",
     "@types/react": "^19.0.10",
     "prettier": "^3.5.1",
-    "tsup": "^8.3.6"
+    "tsup": "^8.4.0"
```

Comment:
Bump tsup so we can use esbuild v0.25 which fixes this [bug](https://github.com/vitejs/vite/issues/18974), that was preventing some Tailwind classes to be properly applied